contracts/extensions/MultisigPermissions.sol

```diff
@@ -439,10 +439,6 @@ contract MultisigPermissions is
     return domainSkillRoleCounts[_domainSkillId][role];
   }
 
-  function getDomainSkillThreshold(uint256 _domainSkillId) public view returns (uint256) {
```

Comment:
Hopefully I'm missing something, but in combination with the commit message, it looks like you deleted this function in order to improve coverage, instead of writing a test for it?

Reply:
The function isn't used anywhere, and the functionality is duplicated in `getDomainSkillRoleThreshold`, so I assumed it was an artifact of an earlier implementation.

Reply:
I mean, it's different. If you get a number back from `getDomainSkillRoleThreshold`, you need access to this function to work out _why_ the number you got back is why it is (i.e. is it set for the this domain specifically vs globally vs based on the number of people with the relevant role in the domain).

That said, I could certainly see this function not being necessary if the block ingestor handles the relevant event where this value is emitted and stores things appropriately for the frontend to access. Generally speaking though, I am uncomfortable with the idea of having storage without corresponding getters.

Reply:
Fair enough